### ISSUE TITLE
DXE-2297 Ordering of `allowedFormats` in `akamai_imaging_policy_image` is inconsistent

### ISSUE THREAD

**Issue by scottwallacesh** [akamai/terraform-provider-akamai#383]
When running a `terraform plan` the ordering of `allowedFormats` is inconsistent between runs and causes false positive reports of changes.

### Terraform Version
```sh
$ terraform version
Terraform v1.3.7
on linux_amd64
+ provider registry.terraform.io/akamai/akamai v3.3.0
```

### Affected Resource(s)
- `akamai_imaging_policy_image`

### Terraform Configuration Files
```hcl
resource "akamai_imaging_policy_image" "Default_Policy" {
  activate_on_production = false
  contract_id            = "C-ABC123"
  json                   = file("policy-snippets/main.json")
  policy_id              = ".auto"
  policyset_id           = "some_policy_id"
}
```
`policy-snippers/main.json`:
```json
    [...]
  "output": {
    "perceptualQuality": "mediumHigh",
    "allowedFormats": [
      "jpeg",
      "webp",
      "avif",
      "png",
      "gif"
    ]
  },
    [...]    
```

### Expected Behavior
`No changes. Your infrastructure matches the configuration.`

### Actual Behavior
Run one:
```diff
                  ~ allowedFormats    = [
                      - "webp",
                        "jpeg",
                      - "png",
                      + "webp",
                        "avif",
                      + "png",
                        "gif",
                    ]
```

Run two:
```diff
                  ~ allowedFormats    = [
                      - "jpeg",
                      - "gif",
                      - "avif",
                        "webp",
                      + "jpeg",
                        "png",
                      + "avif",
                      + "gif",
                    ]
```

Run three:
```text
No changes. Your infrastructure matches the configuration.
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

**Comment by wzagrajcz** [comment]
Hello @scottwallacesh 
Thank You for opening the issue. We will work on recreating the issue and go back to You in case some questions or updates.

Best regards,
Wojciech

**Comment by wzagrajcz** [comment]
Hello @scottwallacesh 
We were able to reproduce this issue and will fix it. 
One more side question. In Your example You use `avif` format which (after brief check) should not be accepted at all. Was that (including the `avif`) config successfully applied?

Best regards,
Wojciech

**Comment by scottwallacesh** [comment]
Yeah, the `avif` value didn't seem to be a problem.  I've not tried to apply recently.

**Comment by scottwallacesh** [comment]
I can confirm this is resolved with the v3.6.0 release.
